_posts/2015-09-30-september-newsletter.md

```diff
@@ -0,0 +1,286 @@
+of months ago we received a [pr][xdg-pr1] from @Yamakaky which was superseded by
+@jck in a [later pr][xdg-pr2].
+
+Following XDG directory specification will let Neovim users to store configuration
+files such as `.nvimrc` and those under `~/.nvim` in the `~/.config` directory,
+which can be overriden by the `$XDG_CONFIG_HOME` environment variable. The
+specification also states that cache files should be stored in a separate
+directory(`~/.local/share`), which is where files like viminfo(now ShaDa) or
+backup/swap can optionally go.
+
+This change makes it simpler for users to backup and manage their configuration
+since it will be stored with other programs that also follow the specification.
+It also keeps the home directory cleaner
+
+### Faster travis builds
+
+We now use [Travis container-based insfrastructure][travis-container] to run
+Neovim builds, which makes CI builds to start immediately. This was
+[implemented][container-pr] by @fwalch, which also did many other improvements
+to our build infrastructure, allowing developers to receive much faster
+feedback when submitting PRs.
+
+### Quickbuild
+
+@jszakmeister is running a [quickbuild][quickbuild] server in his own
+infrastructure. This gives us a backup CI that double checks Neovim PRs, also
+running tests in FreeBSD which is not covered by travis. Thank you for improving
+Neovim robustness @jszakmeister!
+
+## Third-party development
+
+### Neomake
+
+Did you know that there is an alternative to [syntastic][syntastic] that makes
+use of Neovim asynchronous capabilities? [Neomake][neomake] is the best plugin
+for syntatic checking on Neovim: It is extensible like [syntastic][syntastic]
+and the fact that it uses [job-control][job-control] allows it to perform
+checking in background without blocking the user interface. This is very useful
+for compiled languages that are slower to check(typescript, java, .NET).
+
+The migration from [syntastic][syntastic] is also very trivial, great work
+@benekastah!
+
+### FZF
+
+[fzf][fzf] is a command-line fuzzy finder that thanks to its author(@junnegun,
+the same developer behind [vim-plug][vim-plug]), has great Neovim support
+through a plugin that uses our [builtin terminal emulator][terminal-emulator].
+
+FZF is a great alternative to plugins like [ctrlp][ctrlp]: It really fast and
+has the advantage of running in another process, which can make use of
+multi-core systems and doesn't block Neovim user interface. To see how fast and
+responsive it is, just try running `:FZF` to search for files on the linux
+source tree!
+
+### Deoplete
+
+@Shougo has created [deoplete.nvim][deoplete], an asynchronous completion engine
+written as a [remote-plugin][remote-plugin] that makes use of Neovim async
+capabilities to allow completions to be computed without blocking the user
+interface. @Shougo is the sith lord of Vim plugins, here's a list containing
+some of his previous work:
```

Comment:
I have created deoplete.  Because neovim don't support if_lua yet.  I cannot use neocomplete in neovim.
I think it should be described in the paragraph.